cmd/riff/main.go

```diff
@@ -22,7 +22,7 @@ import (
 	"strings"
 
 	// load credential helpers
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
why only gcp?

Reply:
simplifying the dependency graph, I'll put it back